Software/scan_key/AuthenticationSQL.py

```diff
@@ -24,19 +24,21 @@ def machineAuth(id, cursor):
     for auth in authdata:
         if(auth[machineType] != 1):
             print ("USER IS NOT AUTHORIZED\n")
+            os.system("omxplayer deny.wav &")
```

Comment:
I'm well aware it is possible to be TOO abstract - but I wonder if it wouldn't be helpful down the road to have our own audio wrapper. Might make it easier to swap between .wav files or a Piezo buzzer.